R/kinfitr_telemetry.R

```diff
@@ -0,0 +1,52 @@
+#' @importFrom httr2 request req_headers req_retry req_timeout req_perform
+#' @importFrom httr2 resp_body_json req_body_json
+
+get_url <- "http://54.144.240.214/check/kinfitr/"
+post_url <- "http://54.144.240.214/kinfitr/"
+
+get_telemetry <- function(url = get_url, number_of_records = 0) {
+  # checks to see what's been posted to the url endpoint, should return location,
+  # and any other data that gets put there with send_telemetry
+  req <- request(paste(url, as.character(number_of_records), sep = ""))
```

Comment:
Is there a reason you didn't chain these together? I've not used httr2 before, but usually these kinds of things would be chained together.

Reply:
I think it's because I'm writing things like C or Python functions and not R functions. I'm also wrapping them into line by line b/c my R linter is screaming at me if I go over 80 characters.